controller/src/test/java/io/pravega/controller/store/stream/StreamMetadataStoreTest.java

```diff
@@ -706,5 +713,76 @@ public void retentionSetTest() throws Exception {
         streams = store.getStreamsForBucket(0, executor).get();
         assertTrue(!streams.contains(String.format("%s/%s", scope, stream)));
     }
+
+    @Test
+    public void sizeTest() throws Exception {
```

Comment:
Yes, this is a general comment. I made it here sort of arbitrarily. The comment is about two things:

1- Making the test coverage for a given feature more structured so that we can easily identify when going over the code base
2- Making sure we are really covering a broad surface.